biscuit/test/Spec/Executor.hs

```diff
@@ -17,8 +16,9 @@ import           Auth.Biscuit.Datalog.Executor       (ExecutionError (..),
                                                       defaultLimits,
                                                       evaluateExpression)
 import           Auth.Biscuit.Datalog.Parser         (expressionParser, fact,
-                                                      rule)
+                                                      rule, run)
```

Comment:
It doesn't show up in the diff thereafter ; is this import really useful?

Reply:
Oh, there is no -Wall on the tests, that's why it did not complain. I'll enable it and see what else explodes